packages/components/addon/components/hds/button-set/index.js

```diff
@@ -0,0 +1,15 @@
+import Component from '@glimmer/component';
```

Comment:
if you remove the `get classNames` method below and move the class directly in HBS, you can delete this backing class file.